ivy/functional/frontends/mindspore/ops/function/nn_func.py

```diff
@@ -169,3 +179,8 @@ def avg_pool2d(
         count_include_pad=count_include_pad,
         divisor_override=divisor_override,
     )
+
+
+@to_ivy_arrays_and_back
+def flatten(input, order="C ", *, start_dim=1, end_dim=-1):
```

Comment:
```suggestion
def flatten(input, order="C", *, start_dim=1, end_dim=-1):
```
I think this type actually comes from a typo in the docs 😅 https://www.mindspore.cn/docs/zh-CN/r2.1/api_python/ops/mindspore.ops.flatten.html?highlight=flatten#mindspore.ops.flatten